topi/python/topi/transform.py

```diff
@@ -265,6 +265,7 @@ def take(a, indices, axis=None, mode="clip"):
         Specifies how out-of-bound indices will behave.
         clip - clip to the range (default)
         wrap - wrap around the indices
+        fast - no clip or wrap aroud
```

Comment:
around (boundary should be check by user)

Reply:
Added